src/base/solver.jl

```diff
@@ -3,15 +3,15 @@
     make_optimization_model(
         model::MetabolicModel,
         optimizer;
-        sense = MOI.MAX_SENSE,
+        sense = MAX_SENSE,
     )
 
 Convert `MetabolicModel`s to a JuMP model, place objectives and the equality
 constraint.
 
 Here coupling means inequality constraints coupling multiple variables together.
 """
-function make_optimization_model(model::MetabolicModel, optimizer; sense = MOI.MAX_SENSE)
+function make_optimization_model(model::MetabolicModel, optimizer; sense = MAX_SENSE)
```

Comment:
I removed it from FBA (which should always maximize). But here the function is a bit more technical and I expect that people could technically make minimization models, at which point this would set maximization sense only to be overwritten by sense change later. Not sure if that's good or not. (Also the `sense` parameter here has been around forever, no idea if anyone actually isn't relying on it)

Reply:
this makes sense... :)